### PR TITLE
feat!: actual replies

### PR DESCRIPTION
**Pull request by wojpawlik**

### CODE REVIEW

**Review by MKRhere — CHANGES_REQUESTED** [review]
Why not use `Shorthand<>`? Advantage was if types ever changed in Telegram, Context will automatically follow. And parity with the `send*` methods.

**Comment by wojpawlik** [comment]
I'd need to somehow unpack the array to add `reply_to_message_id`. Not idiomatic.

**Comment by wojpawlik** [comment]
Please remove or adjust `replyWithMarkdown`

**Comment by wojpawlik** [comment]
Also, feel free to try to shoehorn `Shorthand` in, I need to rebase anyway. And if you also can't, let me just merge this as is.

File: package.json [hunk 1]
```diff
@@ -19,25 +19,15 @@
...
-    "./future": {
-      "types": "./future.d.ts",
-      "require": "./future.js",
-      "import": "./future.js"
-    },
     "./types": {
-      "types": "./types.d.ts",
-      "require": "./types.js",
-      "import": "./types.js"
+      "types": "./typings/telegram-types.d.ts"
```
Comment (MKRhere) [inline package.json]: The empty JS files are there so ESLint doesn't complain about importing a missing file when there's only a `d.ts`.

Ref: https://t.me/TelegrafJSChat/67718

**Review by MKRhere — APPROVED** [review]

**Review by MKRhere — CHANGES_REQUESTED** [review]
I meant to request changes! Refer to previous comment.

File: package.json (hunk 1 above)
Comment (wojpawlik) [reply package.json:23]: This is no longer necessary. Telegraf 5 requires Node 18, which supports `package.json` `"exports"`.

**Comment by wojpawlik** [comment]
Rebased. Please remove or adjust `Context::replyWithMarkdown`.

**Review by MKRhere — APPROVED** [review]

**Pull request merged**